src/infrastructure/NodeService.js

```diff
@@ -414,6 +418,16 @@ class NodeService {
 
 		return helper.convertArrayToCSV(formattedData);
 	}
+
+	static isHttpsEnabled = async (apiEndpoint) => {
+		try {
+			await Axios.get(apiEndpoint + '/node/info', { timeout: 3000 });
```

Comment:
why axiox? Fetch is the library used in the sdk. You can add timeouts (not as simple but there is a way)

Reply:
There was few service use Axios as well such as node reward endpoint and statistic service endpoint.